dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherOptions.nlsprops

```diff
@@ -174,9 +191,17 @@ option-desc.archive=\
 #------------------------------\n at 72 chars -- leading tab-----------\n\#
 option-key.include=\
 \ \ \ \ --include=value,value,...
-option-desc.include=\
-\tA comma-delimited list of values. The valid values vary depending on \n\
-\tthe action.
+option-desc.javadump.include=\
+\tA comma-delimited list of values. The valid values are: heap, system \n\
+\tthread.
+option-desc.package.include=\
+\tA comma-delimited list of values. The valid values are: all*, usr,   \n\
+\tminify, runnable. If you specify runnable the resulting package will \n\
+\tbe an executable jar that will run the server. The minify option     \n\
+\twill produce the smallest image possible for the server. The usr     \n\
+\toption creates a package that contains the server, and application   \n\
```

Comment:
+\tminify, runnable. If you specify **runnable,** the resulting package **is**\n\
+\an executable jar that **runs** the server. The minify option     \n\
+\t**produces** the smallest image possible for the server. The usr     \n\
+\toption creates a package that contains the **server and application,**   \n\
+\t**but** not the runtime. The default all option produces a server package\n\